src/VecTargets.jl

```diff
@@ -1,12 +1,16 @@
 module VecTargets
 
-include("bnormals.jl")
 using BSON, Parameters, Distributions, DistributionsAD
-using Random: GLOBAL_RNG, shuffle
+using DifferentiationInterface
+import Distributions: Distribution, dim, rand, logpdf, pdf, VariateForm, Continuous
+using Random: AbstractRNG, rand!, randn!, GLOBAL_RNG, shuffle
+using ReverseDiff
```

Comment:
Is this still needed? Or if it is but all of its uses are qualified, could it be `using ReverseDiff: ReverseDiff`? Just to have less namespace pollution.

Reply:
The ReverseDiff is kept as default for gradient, hessian and hvp, so it is still needed.

> could it be using ReverseDiff: ReverseDiff? Just to have less namespace pollution.

Yeah, agreed, let's avoid pollution here